frontend/apps/marketing/cicd/3-app/deploy.rb

```diff
@@ -301,19 +329,21 @@ def get_stack_output(stack_name, output_key, region)
       "BaseDomainName" => options[:base_domain_name],
       "SubdomainName" => options[:subdomain_name],
       "EnvironmentType" => options[:environment_type],
+      "SiteType" => options[:site_type],
       "ContainerImageHashDigest" => options[:container_image_hash],
       "CloudFrontTLSCertificateArn" => certificate_arn,
       "WebApplicationServerSecretsARN" => options[:web_application_server_secrets_arn],
       "RoleBoundary" => options[:cloudformation_role_boundary]
     }
 
     deploy_stack(
-      options[:stack_name],
-      marketing_site_template_path,
-      marketing_site_stack_parameters,
-      options[:region],
-      options[:role_arn],
-      options[:environment_type]
+      stack_name: options[:stack_name],
+      template_file: marketing_site_template_path,
+      parameters: marketing_site_stack_parameters,
+      region: options[:region],
+      role_arn: options[:role_arn],
+      tags: {'environment-type' => options[:environment_type], 'site-type' => options[:site_type]},
```

Comment:
Missing new line here?

```suggestion
      tags: {'environment-type' => options[:environment_type],
      'site-type' => options[:site_type]},
```

Reply:
I thought this hash was short enough to be a one-liner, but standard multi-line, indented seems better!